modules/programs/fd.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+with lib; {
+  meta.maintainers = [ maintainers.uncenter ];
+
+  options.programs.fd = {
+    enable = mkEnableOption
+      "fd, a simple, fast and user-friendly alternative to {command}`find`";
+
+    enableBashIntegration = mkEnableOption "Bash integration" // {
+      default = true;
+    };
+
+    enableZshIntegration = mkEnableOption "Zsh integration" // {
+      default = true;
+    };
+
+    enableFishIntegration = mkEnableOption "Fish integration" // {
+      default = true;
+    };
+
+    enableIonIntegration = mkEnableOption "Ion integration" // {
+      default = true;
+    };
+
+    enableNushellIntegration = mkEnableOption "Nushell integration" // {
+      default = true;
+    };
```

Comment:
Just noticed that none of these options seem to actually do anything. Perhaps the intent is to use `mkIf` for the various `shellAliases` options?

Reply:
Oops! Thanks for pointing that out - can't believe I didn't notice. I originally based this off of the `eza` module, but now looking at that one it seems that `eza ${args}` is always aliased no matter if `enable___Integration` is enabled, and the `enable___Integration` options are used to alias things like `ls` and `ll` and so on. I think we can just remove all of those options in that case?

Reply:
Yeah, can remove them.